http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml

```diff
@@ -38,4 +38,5 @@
     <include file="changesets/20191204_jupyter_start_script.xml" relativeToChangelogFile="true" />
     <include file="changesets/20200122_cluster_followup_table.xml" relativeToChangelogFile="true" />
     <include file="changesets/20200123_cluster_table_gce_refactor.xml" relativeToChangelogFile="true" />
+    <include file="changesets/20200123_populate_date_accessed.xml.xml" relativeToChangelogFile="true" />
```

Comment:
.xml.xml

Reply:
Also can fix date.. `20200213`

Reply:
multi tasking is hard :)

Reply:
I left the date the same just so it's obvious that we ran the migration on the same date...